decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
```

Comment:
- [opional] Please kindly take note that you're not suppose to capitalize the base decorator, the previous reviewer made  a mistake I think. Please ensure that you correct this for your project to fulfil the aim of this milestone. 🙏 

![Code review](https://github.com/DevaWinner/school-library/assets/103386183/02ccfe89-ffb1-4cf8-9d1b-95f8aad6e840)